plugins/org.jboss.tools.openshift.common.ui/src/org/jboss/tools/openshift/internal/common/ui/utils/UIUtils.java

```diff
@@ -485,7 +485,10 @@ public static ElementTreeSelectionDialog createFileDialog(String selectedFile,
             @Override
             public boolean select(Viewer viewer, Object parentElement, Object element) {
                 return element instanceof IContainer 
-                        || (element instanceof IFile && ((IFile)element).getFileExtension().equals(extension));
+                        || (element instanceof IFile 
+                        		&& extension != null 
```

Comment:
@scabanovich if we look at some web features, for example when you want to upload an image to some web-site, you won't see files without extensions until you select "All files". So i think it must be the same way:
if extension is null (means no restrictions) then all files selected anyway.

If you agree, i'll change the PR